src/System.Private.ServiceModel/tests/Stress/SharedPoolsOfWCFObjects/TestHelpers.cs

```diff
@@ -41,6 +50,37 @@ public static EndpointAddress CreateHttpEndpointAddress()
             return new EndpointAddress(s_httpUrl);
         }
 
+        public static EndpointAddress CreateEndPointDuplexAddress()
+        {
+            return UseHttp ? CreateHttpEndpointDuplexAddress() : CreateNetTcpEndpointDuplexAddress();
+        }
+
+        public static EndpointAddress CreateNetTcpEndpointDuplexAddress()
+        {
+            //string address = "net.tcp://cspod222-04vm4.corp.microsoft.com/WcfService1/Service1.svc";
```

Comment:
Are we leaking an internal name here? Please remove, but probably no need to do a history obliterate here.